themes-default/slim/views/addShows_newShow.mako

```diff
@@ -1,14 +1,36 @@
 <%inherit file="/layouts/main.mako"/>
 <%!
+    import json
+
     from medusa import app
     from medusa.indexers.indexer_api import indexerApi
+    from medusa.indexers.indexer_config import indexerConfig
+
+    from six import iteritems, text_type
 %>
 <%block name="scripts">
 <script type="text/javascript" src="js/add-show-options.js?${sbPID}"></script>
 <script type="text/javascript" src="js/blackwhite.js?${sbPID}"></script>
-<script src="js/lib/frisbee@2.0.4.min.js"></script>
-<script src="js/lib/vue-frisbee.min.js"></script>
-<script src="js/vue-submit-form.js"></script>
+## <script src="js/lib/frisbee@2.0.4.min.js"></script>
+## <script src="js/lib/vue-frisbee.min.js"></script>
+## <script src="js/vue-submit-form.js"></script>
```

Comment:
These should be removed before merging.

Reply:
They're used in other files right? I was mainly talking about the python comment.

Reply:
Ah well, no, they're unused.
`frisbee` (`vue-frisbee`) is only used in `vue-submit-form`, and `vue-submit-form` *was* only used in `newShow`.

Reply:
If they're unused remove the libs as we can always add them back.